OpenRA.Mods.Common/Traits/Render/SupportPowerChargeBar.cs

```diff
@@ -36,13 +37,20 @@ public SupportPowerChargeBar(Actor self, SupportPowerChargeBarInfo info)
 
 		float ISelectionBar.GetValue()
 		{
-			if (!self.Owner.IsAlliedWith(self.World.RenderPlayer))
-				return 0;
-
 			var spm = self.Owner.PlayerActor.Trait<SupportPowerManager>();
```

Comment:
#11179 has already been merged, so please change it here. Don't forget to implement `INotifyOwnerChanged`.